### PR TITLE
Cut 0.6.0 release

### PR DESCRIPTION
**Pull request by lucywyman**
Update changelog and bump version

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,9 +2,9 @@ source "https://rubygems.org"
 
 group :development, :test do
   gem 'rake'
-  gem 'rspec', "~> 3.1.0", :require => false
-  gem 'mocha', "~> 0.10.5", :require => false
-  gem 'puppetlabs_spec_helper', :require => false
+  gem 'rspec', "~> 3.1.0", 			:require => false
```
Comment (bmjen) [inline Gemfile]: Tabs?

File: metadata.json [hunk 2]
```diff
@@ -36,7 +36,11 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.0.0 < 5.0.0"
+    },
+    {
+      "name": "pe",
+      "version_requirement": ">= 3.0.0 < 2015.3.0"
```
Comment (bmjen) [inline metadata.json]: Can you update this to `2015.4.0`?

File: Gemfile [hunk 3]
```diff
@@ -15,4 +15,8 @@ end
 
 gem 'puppet-lint', '>= 1.0.0'
 gem 'puppet-lint-unquoted_string-check', :require => false
+
+if RUBY_VERSION < '2.0'
+	  gem 'mime-types', '<3.0', :require => false
```
Comment (bmjen) [inline Gemfile]: Can this be changed to a 2 space indent?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+- List all PVs in a VG
+- Unit tests for types
+
+####Bugfixes
+- Filesystem type can create journal devices
+- Add persistent and minor parameters to logical volume type
+- Make size_is_minsize usable
+- Add support for older lvm versions
+
+
```
Comment (bmjen) [inline CHANGELOG.md]: Remove extra newline

**Comment by bmjen** [comment]
:+1: squash and we're all set!

**Comment by walkamongus** [comment]
just an FYI on the issue in #135...someone besides myself ran into the issue as well so it may be good to get a fix into the next release. I just mention it because it affects physical_volumes with a default parameter value so its likely others will run into it.

**Pull request merged**